_data/wildfly-categories.yaml

```diff
@@ -0,0 +1,140 @@
+---
+categories:
+  - name: Agroal
+    id: agroal
+    description: Agroal Data Source
+  - name: Batch
+    id: batch
+    description: Jakarta EE Batch
+  - name: Bean Validation
+    id: bean-validation
+    description: Jakarta EE Bean Validation
+  - name: Bootable Jar
+    id: bootable-jar
+    description: Bootable Jar
+  - name: Command Line Interface (CLI)
+    id: cli
+    description: Command Line Interface
+  - name: Cloud
+    id: cloud
+    description: Developing and Running a WildFly application on the cloud
+  - name: User Experience
+    id: user-experience
+    description: Impact how the user experiences working with WildFly
+  - name: Clustering
+    id: clustering
+    description: Clustering, including Infinispan and JGroups
+  - name: Concurrency
+    id: concurrency
+    description: Concurrency
+  - name: Console
+    id: console
+    description: Console
+  - name: WildFly Controller
+    id: controller
+    description: WildFly Controller
+  - name: WildFly Core
+    id: core
+    description: WildFly Core
+  - name: Jakarta EE
+    id: ee
+    description: Jakarta EE
+  - name: Jakarta EJB
+    id: ejb
+    description: Jakarta Entreprise Java Beans (EJB)
+  - name: WildFly Elytron
+    id: elytron
+    description: WildFly Elytron
+  - name: Embedded
+    id: embedded
+    description: Embedded
+  - name: IIOP
+    id: iiop
+    description: Internet InterORB Protocol
+  - name: Jakarta RS
+    id: jaxrs
+    description: Jakarta RESTful Web Services
+  - name: JCA
```

Comment:
Jakarta Connectors

Reply:
I think we should also add 'Datasources' and 'Jakarta Resources'.

Reply:
I'm not sure what you are suggesting.

Should that category be `Jakarta Connectors, Datasources & Resources`?

Reply:
@jmesnil Sorry; I was unclear. I mean we should add two new categories and add them to the relevant proposals. (Adding these is not a blocker for this PR.)

WFLY has a single JCA component and the source tree has a single 'connectors' maven module, but from a user POV conflating all these things is confusing. Jakarta Resources is a separate spec from Jakarta Connectors, and Datasources are a general concept defined in Java SE and not inherently tied to Jakarta Connectors.

Since a proposal can have multiple categories we can make this easier.

Reply:
@jmesnil I have some other feedback that I'll send up as a PR to your branch. I'll include this bit ^^^ as a commit.